### PR TITLE
Refactor/board initialization

### PR DESCRIPTION
**Pull request by indykid**
so i changed the way the board is initialized, specs no longer rely on helpers. 
unsure if this makes it better, and yet to discover if this would help with minimax or not

### CODE REVIEW

**Comment by indykid** [comment]
yet to change the way the board is setup in game specs

File: spec/board_spec.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@
   end
 
   it 'knows that occupied position is not valid' do
-    board.add_move(0, :x)
+    board = Board.new([:x, nil, nil, nil, nil, nil, nil, nil, nil])
```
Comment (jsuchy) [inline spec/board_spec.rb:18]: duplication

File: spec/game_spec.rb [hunk 2]
```diff
@@ -90,45 +86,38 @@
...
 
   it 'gets ui to display board at the end' do
-    ui = FakeUi.new([0, 3, 1, 4, 2])
-    game = game_setup(ui)
+    setup_ui([0, 3, 1, 4, 2])
 
     game.play
 
-    expect(ui.board_display_count).to eq(6)
+    expect(ui).to have_received(:display_board).exactly(6).times
```
Comment (jsuchy) [inline spec/game_spec.rb:93]: expectation doesn't match the description of the test

File: spec/board_spec.rb (hunk 1 above)
Comment (indykid) [inline spec/board_spec.rb:18]: @jsuchy you are looking at an older commit, i refactored this already :)

**Pull request merged**